NewLinearLeiosMiniProtocols.md

```diff
@@ -0,0 +1,593 @@
+## Mini Protocols
```

Comment:
I suspect we should pivot this section to instead be a "demonstration of feasibility". IE we're not necessarily claiming that this is what the wire protocol should be, just that it plausibly could be this without requiring excessive resources.

For example, promise a more concrete follow-up CIP that is intended to pin down exact concrete details.